### DISCUSSION TITLE
Does this project support undici?

### DISCUSSION THREAD

**Discussion by braaar** [traceloop/jest-opentelemetry#31] [Q&A] [answered]
node18+ uses a native fetch library known as `undici`.

[Nock doesn't support it yet](https://github.com/nock/nock/issues/2397), afaik. Have you looked into that for this library?

**Comment by nirga** [comment, marked as answer]
Hey! Our library is agnostic to which library you use to do fetching. We're using `axios` in our tests but you can use `undici` as well.
